src/erlang_ls.erl

```diff
@@ -125,7 +124,7 @@ set(port_old, Port) ->
 
 -spec lager_config() -> ok.
 lager_config() ->
-  LoggingEnabled = application:get_env(?APP, logging_enabled, false),
+  LoggingEnabled = application:get_env(?APP, logging_enabled, true),
```

Comment:
The value in `erlang_ls.app.src` is `false`, though. Do we want this to be `true` or `false` by default?

Reply:
I would prefer it to be true, in general. It makes it easier to ask people for extra info when they report problems, to not have to first take them through a process of getting logging enabled.

Reply:
That was my intention, but missed the `.app.src` entry. Will follow-up.

Reply:
I used to be of a different opinion, but I now think that logging enabled is a good default.